rollup-globals.bzl

```diff
@@ -40,6 +40,7 @@ ROLLUP_GLOBALS = {
     "@material/animation": "mdc.animation",
     "@material/auto-init": "mdc.autoInit",
     "@material/base": "mdc.base",
+    "@material/base/types": "mdc.base.types",
```

Comment:
We should add a comment saying that this UMD module name would not match with anything MDC provides; but we just add this to make the linter happy. This is module resolves to a type-only file anyway.